RELEASE.md

```diff
@@ -0,0 +1,39 @@
+# Release Notes
+
+## Bootstrapping the Docker environment
+
+```bash
+$ docker pull condaforge/linux-anvil
+$ docker run -i -t -v /path/to/conda-recipes:/src `docker images condaforge/linux-anvil:latest -q`
```

Comment:
Realized during our adventures this week that this could actually be:

```sh
docker run -i -t -v /path/to/conda-recipes:/src condaforge/linux-anvil:latest
```

I don't care either way, your call duder.